### PR TITLE
I have a created a profile of usman siddique w/signoff

### LINKED ISSUE #4203: [Community] Member Profile: Usman
**Current Behavior**

@usmandev09 has been a consistent contributor and community member.

**Desired Situation**
Let's recognize @usmandev09 as a contributor and community member by creating a profile on https://layer5.io/community/members.

- GitHub: @usmandev09
- Twitter: 
- LinkedIn: [muhammadusmansiddique](https://www.linkedin.com/in/muhammadusmansiddique/)
- Link to profile picture:

A detailed explanation on how to set up a community member profile can be found in the [CONTRIBUTING.md](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

---

**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Join the [Layer5 Community](https://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by rafiya2003**
**Description**
I have created a profile of Usman Siddique
This PR fixes #4203

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit f45361a39c4d2056ea15cbba64649d8ee70d43f7 at: https://64612da569629c39f275c79f--layer5.netlify.app

**Comment by Nikhil-Ladha** [comment]
@UsmanDev09 looking good?

File: src/collections/members/usman-siddique/index.mdx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+name: Usman Siddique
+position: Contributor
+image_path: ./Usman-Dev.jpg
+github: UsmanDev09 
+linkedin: muhammadusmansiddique
+status: Active
+location: Lahore, Punjab, Pakistan
```
Comment (Nikhil-Ladha) [inline src/collections/members/usman-siddique/index.mdx:8]: It should be Lahore, Pakistan, right @UsmanDev09 ?

Comment (UsmanDev09) [reply src/collections/members/usman-siddique/index.mdx:8]: Yes, correct

**Review by UsmanDev09 — APPROVED** [review]

**Review by hirentimbadiya — CHANGES_REQUESTED** [review]
@rafiya2003 , make the changes that Nikhil Suggested.

**Comment by leecalcote** [comment]
Whoo-hoo! @UsmanDev09 🎈

File: src/collections/members/usman-siddique/index.mdx (hunk 1 above)
Comment (rafiya2003) [reply src/collections/members/usman-siddique/index.mdx:8]: sure

**Review by Nikhil-Ladha — APPROVED** [review]

**Pull request merged**